extension/package.json

```diff
@@ -1450,8 +1450,8 @@
           },
           {
             "id": "dvc.learnMore",
-            "title": "Explore More Resources",
```

Comment:
No matter what content is now, let's not make it a about DVC please.

That page itself could be improved, fixed - it can include videos, blogs, links to docs of course. 

And to some extent it will be always about DVC, but it can and should include some things about the extension.

Reply:
OK. I assumed we meant it as a DVC-specific page since the file is called `dvc-learn-more.md` and links to dvc.org/doc (which is the completion event). ⏳

Reply:
Moved to `more-resources` (a243fdb2).